INSTALL-UBUNTU.md

```diff
@@ -0,0 +1,257 @@
+# Objective
+Install the FediBuzz relay using a virtual machine running Ubuntu Server 22.04.4 LTS. If you are familiar with NixOS/Flakes, then [installing the NixOS module](https://github.com/astro/buzzrelay?tab=readme-ov-file#build) for this is by far the easier route! 
+
+But for those of us unfamiliar with NixOS, here's another option. 
+
+# Server Configuration
+My system consisted of the following:
+
+* Router Configured to forward 80, 443 through a firewall to NGINX Proxy Manager
+* NGINX Proxy Manager then directs traffic based on the incoming domain name to the appropriate server.
+* A ProxMox server with an Ubuntu VM hosting the FediBuzz relay application.
+
+## Hardware
+This could be run very cheaply on your preferred hosting site, or on your own home lab.
+
+* Ubuntu 22.04.4 LTS 
+* 1 Core
+* 1 GB RAM
+
+The official fedi.buzz relay traffics around 330 public instance streams for around 3,500 unique followers (some may have multiple relay requests) as of April 2024 with a similar configuration.
+
+If you're connecting to the fedi.buzz relay and perhaps one or two others on your own relay, this should be more than enough.
+
+One caveat here. FediBuzz has an option for individual users to utilize relays as well by "following" a relay actor, like `@tag-dogsofmastodon@relay.com`. 
+
+If you promote this alternative route and many individuals start connecting to your relay, it will cause more outgoing traffic and queue processing, therefore increasing your hardware requirements.
+
+# Domain Name
+As with most fediverse projects, you're going to need a domain. In this particular instance, I used a subdomain of https://relay.{yourdomain}.
+
+# NGINX Proxy Manager Config 
+You'll need an SSL certificate for at least the sub-domain. NGINX Proxy Manager has this built in, but you may need some additional assistance depending on your configuration.
+
+I needed a wildcard SSL for my domain, and used the PorkBun API to make that work. Here's a [blog post about Cloudflare](https://blog.jverkamp.com/2023/03/27/wildcard-lets-encrypt-certificates-with-nginx-proxy-manager-and-cloudflare/), but the setup is very similar for PorkBun.
```

Comment:
Totally unrelated

Reply:
These were part of my personal notes, as stated in the pull request. It was pertinent to me as one of the steps required to get this project working. 

I've removed this, agreed that it should stay in my personal journal entry of the process.